datafusion/core/src/physical_plan/aggregates/row_hash.rs

```diff
@@ -56,6 +65,28 @@ pub(crate) enum ExecutionState {
 use super::order::GroupOrdering;
 use super::AggregateExec;
 
+/// This encapsulates the spilling state
+struct SpillState {
+    /// If data has previously been spilled, the locations of the
+    /// spill files (in Arrow IPC format)
+    spills: Vec<NamedTempFile>,
+
+    /// Sorting expression for spilling batches
+    spill_expr: Vec<PhysicalSortExpr>,
```

Comment:
`sort_by_expr`?

Reply:
I would prefer to keep `spill_` prefix to be aligned with others like `spill_schema`